doc/source/operations/upgrading-openstack.rst

```diff
@@ -137,6 +137,23 @@ Known issues
   applying package updates. This will happen automatically as a post hook when
   running the ``kayobe overcloud host package update`` command.
 
+* After upgrading OpenSearch to the latest 2023.1 container image, we have seen
+  cluster routing allocation be disabled on some systems. See bug for details:
+  https://bugs.launchpad.net/kolla-ansible/+bug/2085943.
```

Comment:
nit: This should probably be a real rst link

Reply:
I don't think it needs to be, I'm not hyperlinking a word, just pointing to the url.